XWebView/XWVHttpConnection.m

```diff
@@ -105,7 +105,7 @@ - (NSURL *)rootURL {
     NSURL *root;
     if (_delegate && [_delegate respondsToSelector:@selector(documentRoot)]) {
         root = [NSURL fileURLWithPath:_delegate.documentRoot isDirectory:YES];
-        assert(root);
+        NSCAssert(root != nil, @"<XWV> you must set a valid documentRoot");
```

Comment:
I really don't know how the proper way to use `NSAssert` and its variants. 
I get `use of undeclared identifier _cmd` when I try to use it.
I've looked for this error and generally people use `NSCAssert` but I really agree with you after reading documentation with more attention.
So any kind of idea?